web/src/emoji_picker.ts

```diff
@@ -295,14 +311,18 @@ function toggle_reaction(emoji_name, event) {
         return;
     }
 
+    if (current_message_id === null) {
+        return;
+    }
+
     const message_id = current_message_id;
     const message = message_store.get(message_id);
     if (!message) {
         blueslip.error("reactions: Bad message id", {message_id});
         return;
     }
 
-    reactions.toggle_emoji_reaction(message, emoji_name, event);
+    reactions.toggle_emoji_reaction(message, emoji_name);
```

Comment:
This change should have been its own prep commit, since is fixing a (possibly latent) bug, and deserves independent investigation.

Reply:
As best I can tell, this function has never accepted a third parameter; this is a bug introduced in 5bb10036b57bca36d531a74e8b03188d43ec7862.